versions.yaml

```diff
@@ -52,8 +52,19 @@ versions:
         - v1alpha2
         - v1beta1
       contour-operator: v1.22.0
-  - version: v1.21.1
+  - version: v1.21.2
     supported: "true"
+    dependencies:
+      envoy: "1.22.6"
+      kubernetes:
+        - "1.23"
+        - "1.22"
+        - "1.21"
+      gateway-api:
+        - v1alpha2
+      contour-operator: v1.21.2
```

Comment:
assuming we will still release an operator, but that might be a bit of more work to do

Reply:
not sure how out of date CI etc. is there